src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
Svenska Datatermgruppen [seems to promote](http://www.datatermgruppen.se/index.php?option=com_content&view=article&id=89&Itemid=91&obj=f104&uttr=host) using *värd* or *värddator* and they have been the go-to source for other translation projects I've been involved with. Sounds a bit formal to me but I'd just trust their judgement.

IMHO *värddator* sounds better (more specific) of the two.